src/components/calcite-select/calcite-select.scss

```diff
@@ -75,7 +71,8 @@
     box-border
     truncate;
   font-size: var(--calcite-select-font-size);
-  padding: var(--calcite-select-spacing);
+  padding-inline: var(--calcite-select-spacing-inline);
```

Comment:
@jcfranco I think this needs to change. padding `!==` padding-inline. `padding-inline` doesn't set top/bottom whereas padding does. Currently, this isn't right because the previous spacing variable set top/bottom/left/right.

Reply:
I only set `padding-inline` and  left out the top/bottom padding because it was being set to 0 (the default).